src/main/java/org/jenkinsci/plugins/oic/OicCredentials.java

```diff
@@ -48,7 +48,7 @@ public OicCredentials(
 
         if (expiresInSeconds != null && expiresInSeconds > 0) {
             long allowedClockSkewFixed = Util.fixNull(allowedClockSkewSeconds, 60L);
-            this.expiresAtMillis = currentTimestamp + (expiresInSeconds - allowedClockSkewFixed) * 1000;
+            this.expiresAtMillis = (currentTimestamp + expiresInSeconds + allowedClockSkewFixed) * 1000;
```

Comment:
This change seems to be incorrect. `currentTimestamp` is already in millis. Therefore, this line should be `this.expiresAtMillis = currentTimestamp + (expiresInSeconds + allowedClockSkewFixed) * 1000`.